test/metabase/db/liquibase_test.clj

```diff
@@ -50,15 +40,6 @@
        (liquibase/with-liquibase [liquibase (->> (mt/dbdef->connection-details :mysql :db {:database-name "liquibase_test"})
                                                  (sql-jdbc.conn/connection-details->spec :mysql)
                                                  mdb.test-util/->ClojureJDBCSpecDataSource)]
-         (testing "Make sure the first line actually matches the shape we're testing against"
```

Comment:
we now create the `databasechangelog` table during `with-liquibase` call, so it's no longer the 1st migration.

Also we have the next test that checks charset already, so I don't think we really need this test